llms/llama/requirements.txt

```diff
@@ -2,3 +2,4 @@ mlx>=0.8.0
 sentencepiece
 torch
 numpy
+tiktoken==0.4.0
```

Comment:
Necessary to pin? Seems the older tokenizer didn't need it

Reply:
Did the old tokenizer had it already installed ?, I added this specific version because it was listed in the requirements of [Llama3](   https://github.com/meta-llama/llama3/blob/bb55334adcedfa9f5da66d2e1ed64e6f3dbd82ed/requirements.txt#L4)

Reply:
Well adding `tiktoken` is fine but pinning it to `0.4.0` didn't seem necessary